src/services/analytics/gtm.ts

```diff
@@ -48,6 +48,23 @@ export const gtmSetChainId = (chainId: string): void => {
   _chainId = chainId
 }
 
+export const enum AbTest {
+  SAFE_CREATION = 'safe-creation',
+}
+
+let _abTest: AbTest | null = null
```

Comment:
You could init it as `{}`.

Reply:
In accordance with the decided approach so an option is not feasible. `null` will be used for the initial value.